src/lightning_app/core/work.py

```diff
@@ -537,6 +537,14 @@ def _cleanup_calls(calls: Dict[str, Any]):
                         final_statuses.append(status)
                 calls[call_hash]["statuses"] = final_statuses
 
+    def start(self):
+        """Starts LightingWork component via L.CloudCompute."""
+        if self.status.stage == WorkStageStatus.STOPPED:
+            raise Exception("A work can be started only once for now.")
+
+        # This enables to start the run method with a phony input and exit.
+        self.run(Action(method="start"))
```

Comment:
ok, so instead of forcing `parallel=True` `cached=True` around the call, we are overriding the logic in `run_once` to effectively have that result, is that correct?

and we can extend the mechanism to other actions (which may have similar semantics or not, we'll see about that)

Reply:
Yes, this is a skateboard version to enable the feature. We would need a bigger refactor to suport this in a cleaner way, but this unblocks MultiNode example.